community_images/vault/hashicorp/image.yml

```diff
@@ -0,0 +1,55 @@
+name: vault
+official_name: Vault
+official_website: https://www.vaultproject.io/
+source_image_provider: Hashicorp
+source_image_repo: docker.io/hashicorp/vault
+source_image_repo_link: https://hub.docker.com/_/vault
+source_image_readme: https://github.com/hashicorp/docker-vault/blob/master/README.md
+rf_docker_link: rapidfort/vault
+image_workflow_name: vault_hashicorp
+github_location: vault/hashicorp
+report_url: https://frontrow.rapidfort.com/app/community/imageinfo/docker.io%2Fhashicorp%2Fvault
+usage_instructions: |
+  $ helm repo add hashicorp https://helm.releases.hashicorp.com
+
+  # install vault, just replace repository with RapidFort registry
+  $ helm install vault hashicorp/vault
+what_is_text: |
+  Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data
+disclaimer: |
+  Trademarks: This software listing is packaged by RapidFort. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
+docker_links:
+  - "[`1.12.0`, `latest` (latest/Dockerfile)](https://github.com/hashicorp/docker-vault/blob/11a5dff987176c6ab24f76cee14f6dd820f42d96/0.X/Dockerfile)"
+input_registry:
+  registry: docker.io
+  account: library
+repo_sets:
+  - vault:
+      input_base_tag: "1.12.0"
+runtimes:
+  - type: k8s
+    script: k8s_coverage.sh
+    helm_repo: hashicorp/vault
```

Comment:
Please add helm: repo, url, chart as per new format.

Reply:
sorry, I think you mentioned about the new format in weekly meeting but i can't recall the details. i don't see anything on the page https://github.com/rapidfort/community-images/blob/main/image_yml.md either. can you pls remind me again about the new format?

Reply:
helm.repo | helm repo to use for k8s runtime, ex: 'bitnami' | "" | yes
-- | -- | -- | --
helm.repo_url | helm repo URL to use for k8s runtime, ex: 'https://charts.bitnami.com/bitnami' | "" | yes
helm.chart | helm chart to use for k8s runtime, ex: 'nats' | "" | yes